x/fees/keeper/fees.go

```diff
@@ -140,52 +130,26 @@ func (k Keeper) IsFeeRegistered(
 	return store.Has(contract.Bytes())
 }
 
-// GetDeployerFees returns all contracts registered by a deployer as []common.Address
-func (k Keeper) GetDeployerFees(ctx sdk.Context, deployerAddress sdk.AccAddress) []common.Address {
-	feeKeys := []common.Address{}
-	store := ctx.KVStore(k.storeKey)
-	iterator := sdk.KVStorePrefixIterator(
-		store,
-		types.GetKeyPrefixDeployerFees(deployerAddress),
-	)
-	defer iterator.Close()
-
-	for ; iterator.Valid(); iterator.Next() {
-		feeKeys = append(feeKeys, common.BytesToAddress(iterator.Key()))
-	}
-
-	return feeKeys
-}
-
-// SetDeployerFees stores a registered contract inverse mapping
-func (k Keeper) SetDeployerFees(ctx sdk.Context, deployerAddress sdk.AccAddress, contractAddress common.Address) {
-	store := prefix.NewStore(
-		ctx.KVStore(k.storeKey),
-		types.GetKeyPrefixDeployerFees(deployerAddress),
-	)
-	store.Set(contractAddress.Bytes(), []byte("1"))
-}
-
-// DeleteDeployerFees removes a registered contract from a deployer's KVStore of
-// registered contracts
-func (k Keeper) DeleteDeployerFees(ctx sdk.Context, deployerAddress sdk.AccAddress, contractAddress common.Address) {
-	store := prefix.NewStore(
-		ctx.KVStore(k.storeKey),
-		types.GetKeyPrefixDeployerFees(deployerAddress),
-	)
-	store.Delete(contractAddress.Bytes())
+// IsDeployerMapSet checks if a fee contract by deployer address mapping is set
+// in store
+func (k Keeper) IsDeployerMapSet(
+	ctx sdk.Context,
+	deployer sdk.AccAddress,
+	contract common.Address,
+) bool {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixDeployer)
+	key := append(deployer.Bytes(), contract.Bytes()...)
+	return store.Has(key)
 }
 
-// IsDeployerFeesRegistered checks if a contract exists in a deployer's KVStore of
-// registered contracts
-func (k Keeper) IsDeployerFeesRegistered(
+// IsWithdrawMapSet checks if a fee contract by withdrawer address mapping is set
+// in store
+func (k Keeper) IsWithdrawMapSet(
```

Comment:
Should be `IsWithdrawerMapSet` for consistency
```suggestion
func (k Keeper) IsWithdrawerMapSet(
```